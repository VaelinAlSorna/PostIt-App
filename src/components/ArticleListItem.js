import React from 'react';
//import { IoIosHeart, IoIosChatboxes } from "react-icons/io";
import { Link } from 'react-router-dom';


const ArticleListItem = ({ _id, title, createdAt, creator, comments, likes}) => (
    <Link to={`view/${_id}`}>
        <h3>{title}</h3>
        <p>Posted by: {creator} at: {createdAt}</p>
        {/*<p> <IoIosHeart /> {likes.length}</p>*/}
        {/*<p><IoIosChatboxes /> { comments.length }</p>*/}
    </Link>
);

export default ArticleListItem;