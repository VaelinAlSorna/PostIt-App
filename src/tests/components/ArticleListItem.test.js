import React from 'react';
import { shallow } from 'enzyme';
import ArticleListItem from '../../components/ArticleListItem';
import articles from '../fixtures/articlesFixtures';

test('should render ArticleListItem correctly', () => {
    const wrapper = shallow(<ArticleListItem {...articles[0]} />);
    expect(wrapper).toMatchSnapshot();
});