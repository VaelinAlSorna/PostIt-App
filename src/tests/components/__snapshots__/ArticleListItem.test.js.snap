// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ArticleListItem correctly 1`] = `
<Link
  replace={false}
  to="view/1"
>
  <p>
    Breaking news
  </p>
  <p>
    1.1.1970
  </p>
  <p>
     
    <Component />
     
    1
  </p>
  <p>
    <Component />
     
    1
  </p>
</Link>
`;
